include/engine/api/route_api.hpp

```diff
@@ -228,8 +228,11 @@ class RouteAPI : public BaseAPI
 
         std::vector<util::json::Object> annotations;
 
-        if (parameters.annotations_type != RouteParameters::AnnotationsType::None)
+        if (parameters.annotations_type != RouteParameters::AnnotationsType::None || parameters.annotations == true)
         {
+            if (parameters.annotations == true)
+                parameters.annotations_type = RouteParameters::AnnotationsType::All;
```

Comment:
This will break new functionality with a list of values. It will always fallback to `All`